core/cap-0033.md

```diff
@@ -651,10 +717,71 @@ operations[5]:
 where `<FEE>`, `<SEQ_NUM>`, and `<TIME_BOUNDS>` should all be substituted
 appropriately.
 
+### Example: Revoke Sponsorship
+In this example, we demonstrate how a sponsorship can be revoked. Let `S`
+be the sponsoring account. Let `K` be the `LedgerKey` for an entry which
+is currently sponsored by `S`. Then the following transaction achieves the
+desired goal:
+
+```
+sourceAccount: S
+fee: <FEE>
+seqNum: <SEQ_NUM>
+timeBounds: <TIME_BOUNDS>
+memo:
+    type: MEMO_NONE
+operations[0]:
+    sourceAccount: S
+    body:
+        type: UPDATE_SPONSORSHIP
+        ledgerKey: K
+```
+
+where `<FEE>`, `<SEQ_NUM>`, and `<TIME_BOUNDS>` should all be substituted
+appropriately.
+
+### Example: Transfer Sponsorship
+In this example, we demonstrate how a sponsorship can be transferred. Let `S1`
+and `S2` be the sponsoring accounts. Let `K` be the `LedgerKey` for an entry
+which is currently sponsored by `S1`. Then the following transaction achieves
+the desired goal:
+
+```
+sourceAccount: S1
+fee: <FEE>
+seqNum: <SEQ_NUM>
+timeBounds: <TIME_BOUNDS>
+memo:
+    type: MEMO_NONE
+operations[0]:
+    sourceAccount: S2
+    body:
+        type: SPONSOR_FUTURE_RESERVES
+        sponsoredID: S1
+operations[1]:
+    sourceAccount: S1
+    body:
+        type: UPDATE_SPONSORSHIP
+        ledgerKey: K
```

Comment:
It's difficult from looking at these two example transactions in isolation to understand what the update sponsorship is doing. Would it be feasible to give these specific names:

- `REMOVE_SPONSORSHIP`
- `TRANSFER_SPONSORSHIP`

The term transfer is what is used in the document so it sounds like the right vocab to use for the operation.

Reply:
By "these" do you mean the operations? I agree that would clarify intentions, but it would lead to more errors because fundamentally remove and transfer are the same operation. So it is my opinion that adding additional restrictions on which one to use would just lead to people using the wrong one and having their transactions fail.